util/pace/util/grid/stretch_transformation.py

```diff
@@ -0,0 +1,129 @@
+import copy as cp
+from typing import Tuple, Union
+
+import numpy as np
+
+from pace.util import Quantity
+
+
+def direct_transform(
+    *,
+    lon: Quantity,
+    lat: Quantity,
+    stretch_factor: float,
+    lon_target: float,
+    lat_target: float,
+) -> Tuple[Union[np.ndarray, Quantity], Union[np.ndarray, Quantity]]:
+    """
+    The direct_transform subroutine from fv_grid_utils.F90.
+    Takes in latitude and longitude in radians.
+    Shrinks tile 6 by stretch factor in area to increse resolution locally.
+    Then performs translation of all tiles so that the now-smaller tile 6 is
+    centeres on lon_target, lat_target.
+
+    Inputs:
+    - lon in radians
+    - lat in radians
+    - stretch_factor (e.g. 3.0 means that the resolution on
+        tile 6 becomes 3 times as fine)
+    - lon_target in degrees (from namelist)
+    - lat_target in degrees (from namelist)
+
+    Outputs:
+    - lon_transform in radians
+    - lat_transform in radians
+    """
```

Comment:
Can you format the doc string so that it matches the other places in pace?
Should look something like this:
```
        Description of function
        Args:
            x (inout): variable x
            y (in): variable y
        Returns:
            z (out): return variable z    
```